src/main/java/org/opendcs/odcsapi/dao/ApiDaoBase.java

```diff
@@ -137,7 +137,7 @@ public ResultSet doQuery2(String q)
 	* Executes an UPDATE or INSERT query.
 	* Thread safe: internally synchronized on the modify-statement.
 	* @param q the query string
-	* @throws DatabaseException  if the update fails.
+	* @throws DbException  if the update fails.
```

Comment:
I might be worth actually making this DatabaseException to be in line with opendcs itself.

granted likely a thing for a different PR.